tests/tests.rs

```diff
@@ -87,15 +87,15 @@ mod tests {
         .1;
         assert_eq!(
             measurements.get("PCR0").unwrap(),
-            "93a8a6a775cd1e1ab8b6121d1b0ce08e99e0976dabfa40663fa8ea9633421305de18c8f95aa2b82d3feb918fe912c838"
+            "a5a11ad2b3a60117c14b9fe1278b4b902ac0407e76a7b2d7a1f9e60dbacbd8a621acfe080c834c291fd13b348af0c5b1"
```

Comment:
Can you explain why the PCRs changed here? I don't think rebuilding the same container image should ever change the PCRs.

Reply:
I'll have a look at the changes from upstream to determine a viable path forward for updating the linuxkit. Indeed, the PCRs should not change

Reply:
There are 2 changes I see in the `.img` file produced by newer version of linuxkit:
* the output is `gzip`ped with the same set of arguments we used before
* the content of the image changed - there are some duplicate entries for `rootfs/etc/hosts` and `rootfs/etc/resolv.conf`